src/_lib-components/Select/index.js

```diff
@@ -35,41 +37,42 @@ export function Select ({ placeholder, label, items, selectedItem, onChange, isO
               <div className='select-label'>
                 {placeholder}
               </div>
-              <div className='select-trigger' onClick={() => { toggleSelect() }}>
+              <button className='select-trigger' onClick={() => { toggleSelect() }}>
                 <div className='select-trigger-text'>
-                  <div>{selectState === true ? placeholder : selectedItem.label}</div>
+                  <div>{isOpen === true ? placeholder : selectedItem.label}</div>
                 </div>
-                <img className='select-trigger-icon' src={selectState === true ? iconUp : iconDown} alt='' />
-              </div>
+                <img className='select-trigger-icon' src={isOpen === true ? iconUp : iconDown} alt='' />
+              </button>
             </div>
         }
 
         {
           placeholder &&
           !selectedItem &&
-            <div className='select-trigger' onClick={() => { toggleSelect() }}>
+            <button className='select-trigger' onClick={() => { toggleSelect() }}>
               <div className='select-trigger-text'>
                 <div>
                   {placeholder}
                 </div>
               </div>
-              <img className='select-trigger-icon' src={selectState === true ? iconUp : iconDown} alt='' />
-            </div>
+              <img className='select-trigger-icon' src={isOpen === true ? iconUp : iconDown} alt='' />
+            </button>
         }
 
         {
-          selectState === true &&
+          isOpen === true &&
             <div className='select-items'>
               {
                 items.map(function (item, index) {
                   return (
-                    <div
-                      className={`select-item ${selectedItem && selectedItem.value === item.value ? 'selected' : ''}`}
-                      onClick={() => { clickItem(item) }}
-                      key={item.value}
-                    >
-                      <div className='select-item-icon' />
-                      <div className='select-item-text'>{item.label}</div>
+                    <div className='select-item' key={index}>
+                      <RadioButton
+                        onClick={() => { selectItem(item) }}
```

Comment:
Bør denne heller bruke `onChange()` i stedet for `onClick()` for å støtte tastaturmennesker også?

Reply:
Oppdatert.